Plugins/FormatSwift/Plugin.swift

```diff
@@ -34,8 +42,16 @@ struct AirbnbSwiftFormatPlugin {
       context.pluginWorkDirectory.string + "/swiftformat.cache",
       "--swift-lint-cache-path",
       context.pluginWorkDirectory.string + "/swiftlint.cache",
-    ] + additionalArguments
+    ] + argumentExtractor.remainingArguments
 
+    if arguments.contains("--log") {
+      // swiftlint:disable:next no_direct_standard_out_logs
+      print(launchPath, arguments.joined(separator: " "))
```

Comment:
Since we are logging in the plugin and in the underlying tool, can we prefix each log with where it comes from so the log file is easier to parse?

Reply:
Good idea